ui/src/app/certificate_requests/row.tsx

```diff
@@ -88,13 +89,33 @@ export default function Row({ id, csr, certificate, ActionMenuExpanded, setActio
             setActionMenuExpanded(id)
         }
     }
-    const getFieldDisplay = (key: string, field: string | undefined) => (
-        field ? (
+
+    const getFieldDisplay = (label: string, field: string | undefined, compareField?: string | undefined) => {
+        const isMismatched = compareField !== undefined && compareField !== field;
+        return field ? (
             <p>
-                <b>{key}</b>: {field}
+                <b>{label}:</b>{" "}
+                <span style={{ color: isMismatched ? 'red' : 'inherit' }}>
+                    {field}
+                </span>
             </p>
-        ) : null
-    );
+        ) : null;
+    };
+
+    const getExpiryColor = (notAfter?: string): string => {
+        if (!notAfter) return 'inherit';
+
+        const expiryDate = new Date(notAfter);
+        const now = new Date();
+        const oneDayInMillis = 24 * 60 * 60 * 1000;
+        const timeDifference = expiryDate.getTime() - now.getTime();
+
+        if (timeDifference < 0 || timeDifference < oneDayInMillis) {
+            return 'red';
+        } else {
+            return 'green';
```

Comment:
how about doing yellow instead of red for the ones that are expiring in 24 hours? Also how about highlighting them in that color instead of changing the font color? Also we should use the same colors as the [semantic](https://vanillaframework.io/docs/settings/color-settings) colors everywhere else:
green: #0e8420
yellow: #f99b11
red: #c7162b

Reply:
Fixed, thanks

Reply:
I would write `const green = #c7162b` etc. at the top to make it more readable.

Reply:
done